docs/data/maturity.json

```
@@ -240,8 +240,8 @@
   "render": {
     "deploy": "x",
     "area": "Render",
-    "maturity": "alpha",
-    "description": "Skaffold render can produce the transformed manifests as a separate step"
+    "maturity": "beta",
+    "description": "Skaffold render provides hydrated manifests rather than creating Kubernetes resources"
```

This text is a bit confusing and doesn't match the uses elsewhere (IMHO):

```suggestion
    "description": "Produce hydrated Kubernetes manifests that reference the built resources"
```

---

thanks, sorry I was rethinking the wording on this and got a little ahead of myself. yours is better 👍